### DISCUSSION TITLE
How to get url hash parameters ?

### DISCUSSION THREAD

**Discussion by Mounir-Bouacihe** [nuxt/nuxt#24981] [Questions] [answered]
Hello,
I'm asking how to get hash parameter in nuxt3, in server side middleware and api. I got an empty hash in getRequestURL(event).

**Comment by masyoudi** [comment, marked as answer]
hash parameter is never sent to the server
[see this answer](https://stackoverflow.com/questions/9967681/how-to-send-url-with-hash-to-nodejs)

**Reply by Mounir-Bouacihe** [reply]
I can see now why. Thanks for your quick answer, I appreciate it.
